### DISCUSSION TITLE
Additional metadata

### DISCUSSION THREAD

**Discussion by danieldegroot2** [xspanger3770/GlobalQuake#79] [General]
An idea could be to add or link additional metadata / website(s) from e.g. https://www.gdacs.org/ where available.
Based on a screenshot there is a report number added, is this a link? I'm not sure what information the report number on its own would provide. It may be better to display it as generic 'latest report' or similar instead. Example

![image](https://github.com/xspanger3770/GlobalQuake/assets/67521919/909fb6e0-bd22-4f24-aba1-509475abae47)

**Comment by YacineBoussoufa** [comment]
All the data shown in GlobalQuake is unconfirmed preliminary data, therefore it cannot be connected to any verified event by any official Seismic Agency.

The "Report No." indicates the number of re-calculation of the magnitude and/or location by the software and it's not connected to any DYFI.

**Comment by danieldegroot2** [comment]
Thanks for clarifying.
